test/integration/util/util.go

```diff
@@ -103,14 +103,14 @@ func (m *MinishiftRunner) IsCDK() bool {
 }
 
 func (m *MinishiftRunner) EnsureRunning() {
-	if m.GetStatus() != "Running" {
+	if !strings.Contains(m.GetStatus(), "Running") {
```

Comment:
This was the reason we had integration test cases were failing.

Reply:
What do we want to ensure running here? In this case it might be the VM, but OpenShift could be stopped or unavailable.